src/common/utility.ts

```diff
@@ -586,21 +586,21 @@ export class Utility {
             hostName = null;
         }
 
-        if (hostName && /[^a-zA-Z0-9\.\-:]/.test(hostName)) {
-            return "Repository host name can only contain alphanumeric characters or .-:";
+        if (hostName && /[^a-zA-Z0-9\.\-:\${}]/.test(hostName)) {
+            return "Repository host name can only contain alphanumeric characters or .-:${}";

```

Comment:
the wording now is not correct. $ or ${} is for environment variable not repository name